vermouth/__init__.py

```diff
@@ -30,4 +30,8 @@
     DATA_PATH = pkg_resources.resource_filename('vermouth', 'data')
     del pkg_resources
 
+import pbr.version
+__version__ = pbr.version.VersionInfo('vermouth').release_string()
+del pbr
```

Comment:
del pbr.version?

Reply:
`del pbr.version` does not delete `pbr`:

```
>>> import pbr.version
>>> del pbr.version
>>> pbr
<module 'pbr' from '/home/jon/local/anaconda3/lib/python3.6/site-packages/pbr/__init__.py'>
```

Reply:
And does deleting pbr also delete pbr.version?

Reply:
If there is no reference it should get garbage collected.